components/constants/network_constants.cc

```diff
@@ -48,6 +48,8 @@ const char kCRLSetPrefix3[] =
 const char kCRLSetPrefix4[] =
     "*://storage.googleapis.com/update-delta/hfnkpimlhhgieaddgfemjhofmfblmnib"
     "/*crxd";
+const char kCRLSetPrefix5[] =
```

Comment:
Curious we can remove `kCRLSetPrefix3` as `kCRLSetPrefix5` is introduced as `*` is super-set of `*crl-set*`?

Reply:
you are right, changed